minerl/Malmo/Minecraft/src/main/java/com/microsoft/Malmo/Client/MalmoEnvServer.java

```diff
@@ -474,13 +477,18 @@ private synchronized void stepClientSync(String command, Socket socket, DataInpu
             profiler.startSection("clientTick");
 
             // TimeHelper.SyncManager.debugLog("[MALMO_ENV_SERVER] <STEP> Received: " + actions);
```

Comment:
I think in general we have cleaner code when removing old debug statements. In this case they may be helpful for diagnosing new bugs if they appear - I could go either way